src/chatClient.cpp

```diff
@@ -1508,10 +1524,15 @@ void GroupChatRoom::addMember(uint64_t userid, chatd::Priv priv, bool saveToDb)
         {
             it->second->mPriv = priv;
         }
+
+        nameMemberResolved = promise::_Void();
```

Comment:
Why do we use `promise::_Void()` here instead of `mPeers[userid]->nameResolved()`

Reply:
It's true, it's better retrurn always `mPeers[userid]->nameResolved()`